src/libultra_code/sintable.c

```diff
@@ -0,0 +1,1029 @@
+#include <ultra64/types.h>
+
+static s16 sintable[0x400] = {
+	0x0000,
+	0x0032,
+	0x0064,
```

Comment:
did you run format on this? pretty sure it will put it in more of a block shape so that it isnt so long